Modules/_sqlite/connection.c

```diff
@@ -645,14 +667,7 @@ _pysqlite_func_callback(sqlite3_context *context, int argc, sqlite3_value **argv
         Py_DECREF(py_retval);
     }
     if (!ok) {
-        pysqlite_state *state = pysqlite_get_state(NULL);
-        if (state->enable_callback_tracebacks) {
-            PyErr_Print();
-        }
-        else {
-            PyErr_Clear();
-        }
-        sqlite3_result_error(context, "user-defined function raised exception", -1);
+        _pysqlite_result_error(context, "user-defined function raised exception");
```

Comment:
Line is longer than 79 characters. This is also the case for the three other uses of this function.

Reply:
The code for this module ignores the column limit, and the original code of this line exceeded it too.

But it is a good opportunity to fix the style!